### PR TITLE
add first mapping

### PR DESCRIPTION
**Pull request by julialongtin**
https://wearezeta.atlassian.net/browse/SQPIT-737

## Checklist

 - [x] The **PR Title** explains the impact of the change.
 - [x] The **PR description** provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.
 - [x] **changelog.d** contains the following bits of information:
   - [x] A file with the changelog entry in one or more suitable sub-sections. The sub-sections are marked by directories inside `changelog.d`.

### CODE REVIEW

**Comment by mythsunwind** [comment]
Before merging this into `develop` can someone please merge the PR https://github.com/wireapp/wire-server/pull/1960 into this one. The PR includes the subsystem mapping (@S1, @S2...) and was based on the `test_mappings` branch.

File: services/brig/test/integration/API/User/Auth.hs [hunk 1]
```diff
@@ -361,6 +361,10 @@ testSendLoginCode brig = do
   let _timeout = fromLoginCodeTimeout <$> responseJsonMaybe rsp2
   liftIO $ assertEqual "timeout" (Just (Code.Timeout 600)) _timeout
 
+-- The testLoginFailure test conforms to the following testing standards:
+-- @SF.Provisioning @TSFI.RESTfulAPI @S2
+
+-- | Test that trying to log in with a wrong password or non-existent email fails.
```
Comment (fisx) [inline services/brig/test/integration/API/User/Auth.hs:367]:
```suggestion
-- The testLoginFailure test conforms to the following testing standards:
-- @SF.Provisioning @TSFI.RESTfulAPI @S2
--
-- Test that trying to log in with a wrong password or non-existent email fails.
```

File: services/brig/test/integration/API/User/Auth.hs [hunk 2]
```diff
@@ -395,6 +399,9 @@ testThrottleLogins conf b = do
     threadDelay (1000000 * (n + 1))
   login b (defEmailLogin e) SessionCookie !!! const 200 === statusCode
 
+-- The testLimitRetries test conforms to the following testing standards:
+-- @SF.Provisioning @TSFI.RESTfulAPI @S2
+
```
Comment (fisx) [inline services/brig/test/integration/API/User/Auth.hs:404]:
```suggestion
```

File: services/brig/test/integration/API/User/Auth.hs [hunk 3]
```diff
@@ -566,6 +573,10 @@ testNoUserSsoLogin brig = do
 -------------------------------------------------------------------------------
 -- Token Refresh
 
+-- The testInvalidCookie test conforms to the following testing standards:
+-- @SF.Provisioning @TSFI.RESTfulAPI @S2
+
+-- | Test that invalid and expired tokens do not work.
```
Comment (fisx) [inline services/brig/test/integration/API/User/Auth.hs:579]:
```suggestion
-- The testInvalidCookie test conforms to the following testing standards:
-- @SF.Provisioning @TSFI.RESTfulAPI @S2
--
-- Test that invalid and expired tokens do not work.
```

File: services/brig/test/integration/API/User/Auth.hs [hunk 4]
```diff
@@ -898,6 +909,9 @@ testRemoveCookiesByLabelAndId b = do
   let lbl = cookieLabel c4
   listCookies b (userId u) >>= liftIO . ([lbl] @=?) . map cookieLabel
 
+-- The testTooManyCookies test conforms to the following testing standards:
+-- @SF.Provisioning @TSFI.RESTfulAPI @S2
+
```
Comment (fisx) [inline services/brig/test/integration/API/User/Auth.hs:914]:
```suggestion
```

File: services/brig/test/integration/API/User/Client.hs [hunk 5]
```diff
@@ -416,6 +416,9 @@ testMultiUserGetPrekeysQualified brig opts = do
       const 200 === statusCode
       const (Right $ expectedUserClientMap) === responseJsonEither
 
+-- The testTooManyClients test conforms to the following testing standards:
+-- @SF.Provisioning @TSFI.RESTfulAPI @S2
+
```
Comment (fisx) [inline services/brig/test/integration/API/User/Client.hs:421]:
```suggestion
```

File: services/brig/test/integration/API/User/Client.hs [hunk 6]
```diff
@@ -436,6 +439,9 @@ testTooManyClients opts brig = do
       const (Just "too-many-clients") === fmap Error.label . responseJsonMaybe
       const (Just "application/json;charset=utf-8") === getHeader "Content-Type"
 
+-- The testRemoveClient test conforms to the following testing standards:
+-- @SF.Provisioning @TSFI.RESTfulAPI @S2
+
```
Comment (fisx) [inline services/brig/test/integration/API/User/Client.hs:444]:
```suggestion
```

File: services/brig/test/integration/API/User/Client.hs [hunk 7]
```diff
@@ -475,6 +481,9 @@ testRemoveClient hasPwd brig cannon = do
           newClientCookie = Just defCookieLabel
         }
 
+-- The testRemoveClientShortPwd test conforms to the following testing standards:
+-- @SF.Provisioning @TSFI.RESTfulAPI @S2
+
```
Comment (fisx) [inline services/brig/test/integration/API/User/Client.hs:486]:
```suggestion
```

File: services/brig/test/integration/API/User/Client.hs [hunk 8]
```diff
@@ -661,6 +670,9 @@ testMissingClient brig = do
     const ["text/plain;charset=utf-8"]
       === map snd . filter ((== "Content-Type") . fst) . responseHeaders
 
+-- The testAddMultipleTemporary test conforms to the following testing standards:
+-- @SF.Provisioning @TSFI.RESTfulAPI @S2
+
```
Comment (fisx) [inline services/brig/test/integration/API/User/Client.hs:675]:
```suggestion
```

File: services/brig/test/integration/API/User/Handles.hs [hunk 9]
```diff
@@ -66,6 +66,10 @@ tests _cl _at conf p b c g =
       test p "GET /users/by-handle/<domain>/<handle> : no federation" $ testGetUserByQualifiedHandleNoFederation conf b
     ]
 
+-- The next line contains a mapping from the testHandleUpdate test to the following test standards:
+-- @SF.Provisioning @TSFI.RESTfulAPI @S2
+
+-- | Test changes to the user's handle.
```
Comment (fisx) [inline services/brig/test/integration/API/User/Handles.hs:72]:
```suggestion
-- The next line contains a mapping from the testHandleUpdate test to the following test standards:
-- @SF.Provisioning @TSFI.RESTfulAPI @S2
--
-- Test changes to the user's handle.
```

File: services/spar/test-integration/Test/Spar/Scim/AuthSpec.hs [hunk 10]
```diff
@@ -147,6 +147,7 @@ testNumIdPs = do
   createToken_ owner (CreateScimToken "drei" (Just defPassword)) (env ^. teSpar)
     !!! checkErr 400 (Just "more-than-one-idp")
 
+-- @SF.Provisioning @TSFI.RESTfulAPI @S2
 -- | Test that a token can only be created as a team owner
```
Comment (fisx) [inline services/spar/test-integration/Test/Spar/Scim/AuthSpec.hs:151]:
```suggestion
-- @SF.Provisioning @TSFI.RESTfulAPI @S2
-- Test that a token can only be created as a team owner
```

File: services/spar/test-integration/Test/Spar/Scim/UserSpec.hs [hunk 11]
```diff
@@ -466,6 +466,9 @@ testExternalIdIsRequired = do
   createUser_ (Just tok) user' (env ^. teSpar)
     !!! const 400 === statusCode
 
+-- The next line contains a mapping from this test to the following test standards:
+-- @SF.Provisioning @TSFI.RESTfulAPI @S2
+
 -- | Test that user creation fails if handle is invalid
```
Comment (fisx) [inline services/spar/test-integration/Test/Spar/Scim/UserSpec.hs:472]:
```suggestion
--
-- Test that user creation fails if handle is invalid
```

**Review by fisx — CHANGES_REQUESTED** [review]

**Review by fisx — APPROVED** [review]

**Pull request merged**